internal/provisioner/utility_group.go

```diff
@@ -132,6 +132,12 @@ func (group utilityGroup) DestroyUtilityGroup() error {
 
 // UpgradeUtilityGroup reapplies the chart for the UtilityGroup. This will cause services to upgrade to a new version, if one is available.
 func (group utilityGroup) UpgradeUtilityGroup() error {
+	logger := group.provisioner.logger.WithField("helm-init", "UpgradeUtilityGroup")
+	err := helmInit(logger, group.kops)
+	if err != nil {
+		logger.WithError(err).Errorf("couldn't re-initialize Helm in the cluster")
```

Comment:
nit:
```suggestion
		logger.WithError(err).Error("Couldn't re-initialize Helm in the cluster")
```

Reply:
I don't know why I'm constantly doing this :joy:

Reply:
lol, me either. My linter complains about this stuff. Maybe you can turn that on for yours somehow?

Reply:
What linter are you using? `golint` and `gopls` complain about a number of things but not this.

Reply:
Lately I have been using https://github.com/golangci/golangci-lint